etc/upversion.txt

```diff
@@ -0,0 +1,101 @@
+# Manually maintained list of all files in tests/*.tests/*.json
+# consumed by bin/upversion when upgrading versions
+#
+# Syntax: ACTION FILE [COMMENT]
+#
+# ACTIONS:
+#   y - update the top level "version" in the files
+#   n - do not update the file
+#
+n tests/event_discovery.tests/empty.json
+n tests/event_system.tests/empty.json
+n tests/event_pointset.tests/empty.json
+n tests/state.tests/empty.json
+n tests/config.tests/empty.json
+n tests/config.tests/delta_x1_gateway.json
+n tests/config.tests/delta_x1_target.json
+n tests/metadata.tests/empty.json
+n tests/state.tests/delta_x1_gateway.json
+n tests/state.tests/delta_x1_target.json
+n tests/configuration_endpoint.tests/simple.json
+n tests/configuration_endpoint.tests/delta.json
+n tests/model_pointset.tests/example.json
+n tests/envelope.tests/lgtw.json
+n tests/envelope.tests/empty.json
+n tests/envelope.tests/errors2.json
+n tests/envelope.tests/example.json
+n tests/envelope.tests/gateway.json
+n tests/envelope.tests/example2.json
+n tests/envelope.tests/errors1.json
+n tests/state.tests/makemodel_upgrade.json #tests message upgrade from v 1
```

Comment:
space after #

Is the # even necessary, or just convenience?  What happens if there's other words after the file... ? Seems like having the # is better overall, just kinda curious... ?

Reply:
Used it to improve readability, but is not necessary,  passed through bash `read action file comment` so everything after the second space is a comment